app/components/ticket-container.tsx

```diff
@@ -1,10 +1,8 @@
 import React from "react";
 
```

Comment:
Removed the ticket edges. Might return to this concept later, but they looked kind of crummy at mobile sizes.